pkg/flowaggregator/s3uploader/s3uploader.go

```diff
@@ -0,0 +1,638 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package s3uploader
+
+import (
+	"bytes"
+	"compress/gzip"
+	"context"
+	"crypto/rand"
```

Comment:
you could just use `math/rand` here, this is not for crypto purposes and `math/rand` is easier to use